pkg/render/node.go

```diff
@@ -1302,12 +1301,10 @@ func (c *nodeComponent) nodeVolumeMounts() []corev1.VolumeMount {
 	if c.vppDataplaneEnabled() {
 		nodeVolumeMounts = append(nodeVolumeMounts, corev1.VolumeMount{MountPath: "/usr/local/bin/felix-plugins", Name: "felix-plugins", ReadOnly: true})
 	}
-	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
-		extraNodeMounts := []corev1.VolumeMount{
-			{MountPath: "/var/log/calico", Name: "var-log-calico"},
-		}
-		nodeVolumeMounts = append(nodeVolumeMounts, extraNodeMounts...)
-	} else if c.cfg.Installation.CNI.Type == operatorv1.PluginCalico {
+
+	nodeVolumeMounts = append(nodeVolumeMounts, corev1.VolumeMount{MountPath: "/var/log/calico", Name: "var-log-calico"})
+
+	if c.cfg.Installation.CNI.Type == operatorv1.PluginCalico {
```

Comment:
Previously, this code block was only executed for OSS clusters. Now, it will be executed for OSS and Enterprise - is that intended? 

I think it should be OK, but I am not 100% sure.

Also, we have two `if c.cfg.Installation.CNI.Type == operatorv1.PluginCalico {` right next to each other that should be combined now, assuming this one shouldn't be `if variant == Enterprise && c.cfg.Installation.CNI.Type == operatorv1.PluginCalico {`

Reply:
I will change the check as ```if c.cfg.Installation.Variant != operatorv1.TigeraSecureEnterprise && c.cfg.Installation.CNI.Type == operatorv1.PluginCalico```